pytest.ini

```diff
@@ -54,5 +57,8 @@ python_files =
     # Ref: https://docs.pytest.org/en/latest/reference.html#confval-python_files
     # Needed to discover legacy nose test modules:
     Test*.py
-testpaths = test/
```

Comment:
Keep this in testpaths. It's better when it's declarative.

Reply:
I do want to avoid having the discovery in two places, it does confuse us. As long we do not have any bugs is better to use a single location.